static/main.ts

```diff
@@ -532,6 +534,34 @@ function getDefaultLangId(subLangId: LanguageKey | undefined, options: CompilerE
     return defaultLangId;
 }
 
+function hideShortlinkInfoButton() {
+    const div = $('.shortlinkInfo');
+    div.hide();
+}
+
+function showShortlinkInfoButton() {
+    const div = $('.shortlinkInfo');
+    div.show();
+}
+
+function initShortlinkInfoButton() {
+    const button = $('.shortlinkInfo div');
+    if (options.metadata && options.metadata['ogCreated']) {
+        button.popover({
```

Comment:
If I understand the code correctly (Can't run locally right now), this needs the user to click on the button to fnid the creation time, and is not shown n the button itself, right?

Oh I guess this way we can add more info in the future now that I think about it! Nice!

Reply:
That was the idea yeah, but also tried alt text. Maybe it can still be plain text, will experiment.